iohub/ngff.py

```diff
@@ -1564,6 +1566,61 @@ def positions(self) -> Generator[tuple[str, Position], None, None]:
         for _, well in self.wells():
             for _, position in well.positions():
                 yield position.zgroup.path, position
+    
+    def rename_wells(self, well_names_csv: str):
+        """Modifies well names (and relevant zattrs) according to old and new paths specified in csv file. 
+
+        CSV input should be in the format:
+
+        oldrow/oldcolumn, newrow/newcolumn
+        
+        For example:
+
+        0/0, 0/A
+        0/1, 0/B
+        0/2, 0/C
+
+        rename_wells will adjust well paths from 0/0 --> 0/A, 0/1  -> 0/B
+
+        """
+        
+        names = []
+
+        with open(well_names_csv, mode='r', newline='', encoding='utf-8') as csvfile:
+            csvreader = csv.reader(csvfile)
+            for row in csvreader:
+                names.append([row[0], row[1]]) 
+
+        names = np.array(names)  ### names is a 2D array of [old_well_path, new_well_path
+
+        modified_wells = []
+        modified_columns = []
+        modified_rows = []
```

Comment:
I think this handling should go to the CLI level. 

Are you saying "skip writing a well name if you've already written to it"? I think this case should live at the CLI level and should print something like "warning: multiple wells were renamed to "B/2"  

Do you handle the case where the well doesn't exist. Maybe this should raise a `ValueError` at the API level (for example, see how Ziwen handles this case in `rename_channel` where `get_channel_index` raises `ValueError` if the name does not exist. 

When this raises a `ValueError`, then the CLI level can catch the exception, print a message to the user, and continue.